### PR TITLE
ci: finish early refactor

### PR DESCRIPTION
**Pull request by karlek**

### CODE REVIEW

File: .circleci/exit-if-path-not-changed.bash [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+set -e
+
+# This script makes CircleCI exit a current job if
+# the paths passed as an argument haven't changed.
+# This is eg used for preventing building native
+# builds unless frontend/native or frontend/shared
+# have changed.
+# 
+# Example usage:
+# ./exit-if-path-not-changed.sh path/to/dir1 path/to/dir2
```
Comment (mewmew) [inline .circleci/exit-if-path-not-changed.bash:11]:
```suggestion
# ./exit-if-path-not-changed.bash path/to/dir1 path/to/dir2
```

File: .circleci/exit-if-path-not-changed.bash [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+# 1. Get all the arguments of the script
+# https://unix.stackexchange.com/a/197794
+PATHS_TO_SEARCH="$*"
+
+# 2. Make sure the paths to search are not empty
+if [ -z "$PATHS_TO_SEARCH" ]; then
+    echo "Please provide the paths to search for."
+    echo "Example usage:"
+    echo "./exit-if-path-not-changed.sh path/to/dir1 path/to/dir2"
```
Comment (mewmew) [inline .circleci/exit-if-path-not-changed.bash:26]:
```suggestion
    echo "./exit-if-path-not-changed.bash path/to/dir1 path/to/dir2"
```

Or rename the script to `.sh`

**Review by mewmew — APPROVED** [review]
LGTM. Really happy to see the CI getting faster and faster!

Gotta go fast!

**Comment by karlek** [comment]
I'm adding `bash` to our base image to fix the build problem

**Comment by mewmew** [comment]
Time for me to sleep. See you tomorrow bror!

**Pull request merged**